src/main/java/assets/game.js

```diff
@@ -25,6 +27,20 @@ function makeGrid(table, isPlayer, gridSize) {
     }
 }
 
+function canMoveCheck()
```

Comment:
I like breaking this out into it's own function, this is something I know I should've done more.